app/javascript/packs/modal.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener('DOMContentLoaded', function(){
+  var buttons = document.querySelectorAll(".btn");
+  var closes = document.querySelectorAll(".modal-close");
+  
+  function openModal(event) {
+    var modal = document.querySelector("div[id="+event.currentTarget.id+"]");
+    modal.classList.add('opacity-100');
+    modal.classList.add('z-50');
+  }
+  
+  function closeModal(event) {
+    var modal = document.querySelector("div[id="+event.currentTarget.id+"]");
+    modal.classList.remove('opacity-100');
+    modal.classList.remove('z-50');
```

Comment:
the same about class here

Reply:
We are using the TailwindCSS tool, that's why we put it that way, since we are just learning to handle its classes.